src/commands/protocols/common/simple-asset-sync-command.js

```diff
@@ -0,0 +1,176 @@
+import { setTimeout } from 'timers/promises';
+import Command from '../../command.js';
+import {
+    COMMAND_RETRIES,
+    ERROR_TYPE,
+    OPERATION_ID_STATUS,
+    OPERATION_STATUS,
+    SIMPLE_ASSET_SYNC_PARAMETERS,
+} from '../../../constants/constants.js';
+
+class SimpleAssetSyncCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.tripleStoreService = ctx.tripleStoreService;
+        this.ualService = ctx.ualService;
+        this.operationIdService = ctx.operationIdService;
+        this.getService = ctx.getService;
+
+        this.errorType = ERROR_TYPE.COMMIT_PROOF.SIMPLE_ASSET_SYNC_ERROR;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        const {
+            operationId,
+            blockchain,
+            contract,
+            tokenId,
+            keyword,
+            hashFunctionId,
+            epoch,
+            assertionId,
+            stateIndex,
+        } = command.data;
+
+        await this.operationIdService.updateOperationIdStatus(
+            operationId,
+            blockchain,
+            OPERATION_ID_STATUS.COMMIT_PROOF.SIMPLE_ASSET_SYNC_START,
```

Comment:
It will be interesting to monitor the number of asset sync events to have an idea of how many commits are missing due to dialing issues.